engine/polyglot-api/src/main/scala/org/enso/polyglot/runtime/Runtime.scala

```diff
@@ -1485,6 +1489,10 @@ object Runtime {
         ")"
     }
 
+    /** A notification from the server confirming opening a file.
+      */
+    final case object OpenedFileNotification extends ApiResponse
```

Comment:
If we now expect a response from `Api.OpenFileNotification`, they should be called `OpenFileRequest` and `OpenFileResponse` so that the naming matches the rest of the messages. Messages named `*Notification` means that we don't expect a response.

Reply:
Done